git-gud.1

```diff
@@ -0,0 +1,70 @@
+.TH GIT-GUD 1 "19 Feb 2020" "version 0.0"
+.SH NAME
+git-gud \- get good at git with git-gud
+.SH SYNOPSIS
+.B git gud
+[\-\-version] [\-\-help]  
+<command> [<args>]
+.SH DESCRIPTION
+.B Git Gud
```

Comment:
This is showing a weird space because of the bold. The git manpage doesn't have "Git" bolded, so I think it'd make sense to have this not bolded.

What do you think?

Reply:
This is not true. The bold text does not result in a spacing issue that is just the justification that man pages use. I did make an unrelated minor change to the spacing somewhere else though. Also, most man pages I have seen (and I read at least one man page an hour at this point) have the name of the program emboldened, including Vim and the terminal emulator I use.